Library/Formula/dos2unix.rb

```diff
@@ -13,8 +13,8 @@ class Dos2unix < Formula
   end
 
   devel do
-    url "http://waterlan.home.xs4all.nl/dos2unix/dos2unix-7.3-beta3.tar.gz"
-    sha256 "4b34d0aa81891795566982af901a84efd54344e1e1bcba138329eb5afe0fdc68"
+    url "http://waterlan.home.xs4all.nl/dos2unix/dos2unix-7.3.1-beta5.tar.gz"
```

Comment:
Looks like it scanned OK: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/32900/version=mountain_lion/testReport/brew-test-bot/mountain_lion/install___devel_dos2unix/